src/components/data/DataTable/DataTable.stories.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import _ from 'lodash';
```

Comment:
optional: using explicit import instead global variables is one of our current refactors in UI https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/2148073473/Ongoing+Refactors+in+Cloudify+UI#Using-explicit-imports-in-widgets :) 
```suggestion
import { map } from 'lodash';
```

Reply:
thank you! good one

Reply:
could I ask where do you getting errors/issues? 🤔 because I tried to run storybook locally after changing lodash imports and replacing `_.map` to `map` and I don't see any issue, but maybe I didn't notice something. It would be useful information for future migration work as well :)

Reply:
and thanks for adjustments in other places 🎉